metrics/README.md

```diff
@@ -0,0 +1,60 @@
+# Clear Containers metrics tests
+
+This is a collection of scripts designed to measure different aspects
+and components which forms Clear Containers.
+
+## Metrics tests - content
+
+The tests are organized in the following sections:
+
+| Section   | Short description
+| ----------| --------------------------------------------------------------------------------
+| density   | Masure the amount of memory used by N containers under different configurations.
+| network   | Measere a set of network essentials such as: bandwidth, jitter, latency etc.
+| storage   | Measure storage bandwith using different configurations.
+| time      | Measure the latency that a container[s] take[s] to do a determined workload.
+
+## HOWTO execute metrics scripts
+
+This section includes a helper script `run_all_metrics.sh`. It allows to execute all metrics
+tests or only a specific section.
+
+## Options
+
+if no options are specified, `run_all_metrics.sh` will execute all tests by default.
+
+### Only run the metrics tests from time section
+```
+-l, --latency  Run latency/time metrics tests
```

Comment:
This is network latency only right. Or is this boot latency also?

Reply:
well that executes metrics tests that measure the latency about a container takes to do a determined workload/task